observers/run-observations.sh

```diff
@@ -264,6 +264,7 @@ run_scheduled_recipes() {
   run_recipe_if_needed "recipe-projects.yaml" "morning" "PROJECTS.md" "weekday-only"
   run_recipe_if_needed "recipe-work-personal.yaml" "evening" ".work-personal"
   run_recipe_if_needed "recipe-interactions.yaml" "hourly" "INTERACTIONS.md"
+  run_recipe_if_needed "recipe-chrome-history.yaml" "4h" "CHROME_HISTORY.md" "weekday-only"
```

Comment:
Wasn't sure how frequently this should run or where this should get placed within the list of observers. If it's fine, we can leave it be, but i'm open to changing it as well.

Reply:
yeah I think that is fine